packages/react-ui-components/src/MultiSelectBox_ListPreviewSortable_DraggableListPreviewElement/multiSelectBox_ListPreviewSortable_DraggableListPreviewElement.js

```diff
@@ -127,20 +134,22 @@ export default class MultiSelectBox_ListPreviewSortable_DraggableListPreviewElem
                             icon={'grip-lines-vertical'}
                             className={theme.selectedOption__moveButton}
                             hoverStyle={'clean'}
-                            />
+                        />
                     )}
-                    <div className={theme.selectedOptions__innerPreview}>
+                    <div className={theme.selectedOptions__innerPreview} role="button" onClick={() => {
```

Comment:
Not every option has an uri (not every reference is a document) and this would create an invalid action. So you need to check if an uri exists. This is another reason why the ReferenceEditor should supply the handling as it knows more about the options themselves.